wpeverest/everest-forms
# Fix - Success message issue with default form due to redirection.

### All Submissions:






### Changes proposed in this Pull Request:



There was an error with smart tags in form submit success message. This issue has been fixed in this PR.

### How to test the changes in this Pull Request:

1. Install new  WordPress.
2. Activate Everest 
3. Submit the default form.
4. Ensure success message working.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Success message issue with default form due to redirection.
